CONTRIBUTING.md

```diff
@@ -53,11 +53,9 @@ For features that you have a proposed solution for, please write an [RFC](./rfcs
 
 When submitting a pull request, please ensure your branch meets the following criteria:
 
-* It is a passing build (`gulp test --build` returns success).
```

Comment:
Can we direct people to somewhere in the README that explains how to run the tests?

Reply:
Good idea. I'll get that added to this file.